libs/design-system/src/ThemeProvider.tsx

```diff
@@ -56,10 +56,6 @@ const notificationStyles = (theme: MantineTheme) => {
   };
 };
 
-declare module '@mantine/core' {
-  export type MantineColor = MantineColor | 'gradient';
-}
-
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Why did we need to remove this? I think this lessens type safety when using the `color`/`c` prop on Mantine components‏

Reply:
TS was complaining as it's a recursive type definition. MantineColor wasn't imported anywhere in this file.